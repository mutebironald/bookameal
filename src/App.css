@import "../node_modules/bootstrap/dist/css/bootstrap.css";
@import "../node_modules/toastr/build/toastr.min.css";

body{
  background-color: coral;
}
.meals-container{
  width:50%;
}


.meal-data{
  padding:10px;
  padding-left:5%;
  
}

.radio{
  display: inline
}

.input-wrapper > input {
  display: inline !important;
}
.center-div {
  margin: 5% auto 0 auto;
  width: 45%;
}
.error {
  color: red;
  text-align-last: justify
}

div.meals{
  float: right;
}


/* Style the tab */
.tab {
  float: left;
  border: 1px solid #ccc;
  background-color: #f1f1f1;
  width: 30%;
  height: 300px;
}

/* Style the buttons that are used to open the tab content */
.tab button {
  display: block;
  background-color: inherit;
  color: black;
  padding: 22px 16px;
  width: 100%;
  border: none;
  outline: none;
  text-align: left;
  cursor: pointer;
  transition: 0.3s;
}

/* Change background color of buttons on hover */
.tab button:hover {
  background-color: #ddd;
}

/* Create an active/current "tab button" class */
.tab button.active {
  background-color: #ccc;
}

/* Style the tab content */
.tabcontent {
  float: left;
  padding: 0px 12px;
  border: 1px solid #ccc;
  width: 70%;
  border-left: none;
  height: 300px;
}


/* Dropdown Button */
.dropbtn {
  background-color: #4CAF50;
  color: white;
  padding: 16px;
  font-size: 16px;
  border: none;
}

/* The container <div> - needed to position the dropdown content */
.dropdown {
  position: relative;
  display: inline-block;
}

/* Dropdown Content (Hidden by Default) */
.dropdown-content {
  display: none;
  position: absolute;
  background-color: #f1f1f1;
  min-width: 160px;
  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
  z-index: 1;
}

/* Links inside the dropdown */
.dropdown-content a {
  color: black;
  padding: 12px 16px;
  text-decoration: none;
  display: block;
}

/* Change color of dropdown links on hover */
.dropdown-content a:hover {background-color: #ddd;}

/* Show the dropdown menu on hover */
.dropdown:hover .dropdown-content {display: block;}

/* Change the background color of the dropdown button when the dropdown content is shown */
.dropdown:hover .dropbtn {background-color: #3e8e41;}
