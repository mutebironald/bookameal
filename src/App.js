import React from 'react';
import logo from './logo.svg';
import './App.css';
import SignupForm from './components/users/SignupForm';
import LoginForm from './components/users/LoginForm';
import AddMealForm from './components/meals/AddMealForm';
import UpdateMealForm from './components/meals/UpdateMealForm';
import GetMeals from './components/meals/GetMeals';
import GetMealPage from './components/meals/GetMealPage';
import DeleteMeal from './components/meals/DeleteMeal';
import GetMenu from './components/menus/GetMenu';
import UserMenu from './components/menus/UserMenu';
import CreateMenuPage from './components/menus/CreateMenuPage';
import CreateOrder from './components/orders/CreateOrder';
import GetOrders from './components/orders/GetOrders'
import AdminOrders from './components/orders/AdminOrders';
import AdminMenu from './components/menus/AdminMenu';
import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
import trial from './components/trial';
import Meal from './components/meals/Meal';
import home from './components/home/home'
import LandingPage from './components/LandingPage';
import { ThemeProvider } from 'mineral-ui/themes';
import Whoops404 from './components/./Whoops404';
import AddMealModal from './components/meals/AddMealModal';

import AdminLoginForm from './components/users/AdminLoginForm';
import AdminSignupForm from './components/users/AdminSignupForm';

import trand from './components/trand';
import GetOrderHistory from './components/orders/GetOrderHistory'

import {
  Collapse,
  Navbar,
  NavbarToggler,
  NavbarBrand,
  Nav,
  NavItem,
  NavLink,
  NavDropdown,
  DropdownMenu,
  DropdownItem,
  UncontrolledDropdown,
  DropdownToggle,
  } from 'reactstrap';


import {Switch, Router, Route} from 'react-router-dom';
import history from './history'
import { Provider } from 'react-redux';
import thunk from 'redux-thunk'
import { createStore, applyMiddleware, compose } from 'redux';
import mainReducer from './reducers';

import { getAllMeals } from './actions';



const store = createStore(
  mainReducer,
  compose(
  applyMiddleware(thunk),
  window.devToolsExtension ? window.devToolsExtension() : f => f
  )
);

store.dispatch(getAllMeals());


class App extends React.Component {
  constructor(props){
    super(props);

  this.toggle = this.toggle.bind(this);
  this.state = {
    isOpen: false
  };

  }
  

  toggle() {
    <AddMealModal/>
    this.setState({

      isOpen: !this.state.isOpen
    });

  }

  render() {
    const supportsHistory = 'pushState' in window.history

    return (
      <ThemeProvider theme={{ color_themePrimary: 'dusk' }}>
      <Provider store={store}>
        <Router history={history} forceRefresh={supportsHistory}>
          
              <Switch>
                <Route exact path="/" component={SignupForm}/>
                <Route exact path="/signup" component={SignupForm}/>
                <Route path="/login" component={LoginForm}/>
                <Route path="/addMeal" component={AddMealForm}/>
                <Route path="/updateMeal" component={UpdateMealForm}/>
                <Route path="/getMeals" component={GetMeals}/>
                <Route path="/get" component={GetMealPage}/>
                <Route path="/deleteMeal" component={DeleteMeal}/> 
                <Route path="/getMenu" component={GetMenu}/>
                <Route path="/createMenu" component={CreateMenuPage}/>
                <Route path="/createOrder" component={CreateOrder}/>
                <Route path="/getOrder" component={GetOrders}/>
                <Route path="/test" component={trial}/>
                <Route path="/home" component={home}/>
                <Route path="/Meal" component={Meal}/>
                <Route path="/userMenu" component={UserMenu}/>
                <Route path="/adminOrders" component={AdminOrders}/>
                <Route path="/adminMenu" component={AdminMenu}/>

                <Route path="/adminlogin" component={AdminLoginForm}/>
                <Route path="/adminregister" component={AdminSignupForm}/>


                <Route path="/day" component={LandingPage}/>
                <Route path ="/now" component={trand}/>
                <Route path="/history" component={GetOrderHistory}/>
                <Route component={Whoops404}/>
              </Switch>
        
  </ Router>
 
</ Provider>
</ThemeProvider>


    );
  }
}


export default App;
