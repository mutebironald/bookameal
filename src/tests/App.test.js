import React from 'react';
import  App   from '../App';

describe("App component", () => {
    let wrapper;
    let mockFn;
    beforeEach(() => {
        mockFn = jest.fn();
        wrapper = shallow(<App/>)

    });
    it("renders successfully", () => {
        expect(wrapper).toHaveLength(1);
    });
    
})
