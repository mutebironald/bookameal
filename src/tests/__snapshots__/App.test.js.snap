// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Router
        forceRefresh={true}
        history={
                Object {
                        "action": "POP",
                        "block": [Function],
                        "createHref": [Function],
                        "go": [Function],
                        "goBack": [Function],
                        "goForward": [Function],
                        "length": 1,
                        "listen": [Function],
                        "location": Object {
                          "hash": "",
                          "pathname": "/",
                          "search": "",
                          "state": undefined,
                        },
                        "push": [Function],
                        "replace": [Function],
                      }
        }
>
        <Switch>
                <Route
                        component={[Function]}
                        exact={true}
                        path="/"
                />
                <Route
                        component={[Function]}
                        exact={true}
                        path="/signup"
                />
                <Route
                        component={[Function]}
                        path="/login"
                />
                <Route
                        component={[Function]}
                        path="/addMeal"
                />
                <Route
                        component={[Function]}
                        path="/updateMeal"
                />
                <Route
                        component={[Function]}
                        path="/getMeals"
                />
                <Route
                        component={[Function]}
                        path="/deleteMeal"
                />
                <Route
                        component={[Function]}
                        path="/getMenu"
                />
                <Route
                        component={[Function]}
                        path="/createMenu"
                />
                <Route
                        component={[Function]}
                        path="/createOrder"
                />
                <Route
                        component={[Function]}
                        path="/getOrder"
                />
                <Route
                        component={[Function]}
                        path="/test"
                />
                <Route
                        component={[Function]}
                        path="/home"
                />
                <Route
                        component={[Function]}
                        path="/Meal"
                />
                <Route
                        component={[Function]}
                        path="/userMenu"
                />
                <Route
                        component={[Function]}
                        path="/adminOrders"
                />
                <Route
                        component={[Function]}
                        path="/adminMenu"
                />
                <Route
                        component={[Function]}
                        path="/adminlogin"
                />
                <Route
                        component={[Function]}
                        path="/adminregister"
                />
                <Route
                        component={[Function]}
                        path="/day"
                />
                <Route
                        component={[Function]}
                        path="/now"
                />
                <Route
                        component={[Function]}
                        path="/history"
                />
                <Route
                        component={[Function]}
                />
        </Switch>
</Router>,
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Switch>
          <Route
                    component={[Function]}
                    exact={true}
                    path="/"
          />
          <Route
                    component={[Function]}
                    exact={true}
                    path="/signup"
          />
          <Route
                    component={[Function]}
                    path="/login"
          />
          <Route
                    component={[Function]}
                    path="/addMeal"
          />
          <Route
                    component={[Function]}
                    path="/updateMeal"
          />
          <Route
                    component={[Function]}
                    path="/getMeals"
          />
          <Route
                    component={[Function]}
                    path="/deleteMeal"
          />
          <Route
                    component={[Function]}
                    path="/getMenu"
          />
          <Route
                    component={[Function]}
                    path="/createMenu"
          />
          <Route
                    component={[Function]}
                    path="/createOrder"
          />
          <Route
                    component={[Function]}
                    path="/getOrder"
          />
          <Route
                    component={[Function]}
                    path="/test"
          />
          <Route
                    component={[Function]}
                    path="/home"
          />
          <Route
                    component={[Function]}
                    path="/Meal"
          />
          <Route
                    component={[Function]}
                    path="/userMenu"
          />
          <Route
                    component={[Function]}
                    path="/adminOrders"
          />
          <Route
                    component={[Function]}
                    path="/adminMenu"
          />
          <Route
                    component={[Function]}
                    path="/adminlogin"
          />
          <Route
                    component={[Function]}
                    path="/adminregister"
          />
          <Route
                    component={[Function]}
                    path="/day"
          />
          <Route
                    component={[Function]}
                    path="/now"
          />
          <Route
                    component={[Function]}
                    path="/history"
          />
          <Route
                    component={[Function]}
          />
</Switch>,
        "forceRefresh": true,
        "history": Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Route
              component={[Function]}
              exact={true}
              path="/"
/>,
            <Route
              component={[Function]}
              exact={true}
              path="/signup"
/>,
            <Route
              component={[Function]}
              path="/login"
/>,
            <Route
              component={[Function]}
              path="/addMeal"
/>,
            <Route
              component={[Function]}
              path="/updateMeal"
/>,
            <Route
              component={[Function]}
              path="/getMeals"
/>,
            <Route
              component={[Function]}
              path="/deleteMeal"
/>,
            <Route
              component={[Function]}
              path="/getMenu"
/>,
            <Route
              component={[Function]}
              path="/createMenu"
/>,
            <Route
              component={[Function]}
              path="/createOrder"
/>,
            <Route
              component={[Function]}
              path="/getOrder"
/>,
            <Route
              component={[Function]}
              path="/test"
/>,
            <Route
              component={[Function]}
              path="/home"
/>,
            <Route
              component={[Function]}
              path="/Meal"
/>,
            <Route
              component={[Function]}
              path="/userMenu"
/>,
            <Route
              component={[Function]}
              path="/adminOrders"
/>,
            <Route
              component={[Function]}
              path="/adminMenu"
/>,
            <Route
              component={[Function]}
              path="/adminlogin"
/>,
            <Route
              component={[Function]}
              path="/adminregister"
/>,
            <Route
              component={[Function]}
              path="/day"
/>,
            <Route
              component={[Function]}
              path="/now"
/>,
            <Route
              component={[Function]}
              path="/history"
/>,
            <Route
              component={[Function]}
/>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "exact": true,
              "path": "/",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "exact": true,
              "path": "/signup",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/login",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/addMeal",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/updateMeal",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/getMeals",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/deleteMeal",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/getMenu",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/createMenu",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/createOrder",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/getOrder",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/test",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/home",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/Meal",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/userMenu",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/adminOrders",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/adminMenu",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/adminlogin",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/adminregister",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/day",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/now",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/history",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Router
          forceRefresh={true}
          history={
                    Object {
                              "action": "POP",
                              "block": [Function],
                              "createHref": [Function],
                              "go": [Function],
                              "goBack": [Function],
                              "goForward": [Function],
                              "length": 1,
                              "listen": [Function],
                              "location": Object {
                                "hash": "",
                                "pathname": "/",
                                "search": "",
                                "state": undefined,
                              },
                              "push": [Function],
                              "replace": [Function],
                            }
          }
>
          <Switch>
                    <Route
                              component={[Function]}
                              exact={true}
                              path="/"
                    />
                    <Route
                              component={[Function]}
                              exact={true}
                              path="/signup"
                    />
                    <Route
                              component={[Function]}
                              path="/login"
                    />
                    <Route
                              component={[Function]}
                              path="/addMeal"
                    />
                    <Route
                              component={[Function]}
                              path="/updateMeal"
                    />
                    <Route
                              component={[Function]}
                              path="/getMeals"
                    />
                    <Route
                              component={[Function]}
                              path="/deleteMeal"
                    />
                    <Route
                              component={[Function]}
                              path="/getMenu"
                    />
                    <Route
                              component={[Function]}
                              path="/createMenu"
                    />
                    <Route
                              component={[Function]}
                              path="/createOrder"
                    />
                    <Route
                              component={[Function]}
                              path="/getOrder"
                    />
                    <Route
                              component={[Function]}
                              path="/test"
                    />
                    <Route
                              component={[Function]}
                              path="/home"
                    />
                    <Route
                              component={[Function]}
                              path="/Meal"
                    />
                    <Route
                              component={[Function]}
                              path="/userMenu"
                    />
                    <Route
                              component={[Function]}
                              path="/adminOrders"
                    />
                    <Route
                              component={[Function]}
                              path="/adminMenu"
                    />
                    <Route
                              component={[Function]}
                              path="/adminlogin"
                    />
                    <Route
                              component={[Function]}
                              path="/adminregister"
                    />
                    <Route
                              component={[Function]}
                              path="/day"
                    />
                    <Route
                              component={[Function]}
                              path="/now"
                    />
                    <Route
                              component={[Function]}
                              path="/history"
                    />
                    <Route
                              component={[Function]}
                    />
          </Switch>
</Router>,
        "store": Object {
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Switch>
            <Route
                        component={[Function]}
                        exact={true}
                        path="/"
            />
            <Route
                        component={[Function]}
                        exact={true}
                        path="/signup"
            />
            <Route
                        component={[Function]}
                        path="/login"
            />
            <Route
                        component={[Function]}
                        path="/addMeal"
            />
            <Route
                        component={[Function]}
                        path="/updateMeal"
            />
            <Route
                        component={[Function]}
                        path="/getMeals"
            />
            <Route
                        component={[Function]}
                        path="/deleteMeal"
            />
            <Route
                        component={[Function]}
                        path="/getMenu"
            />
            <Route
                        component={[Function]}
                        path="/createMenu"
            />
            <Route
                        component={[Function]}
                        path="/createOrder"
            />
            <Route
                        component={[Function]}
                        path="/getOrder"
            />
            <Route
                        component={[Function]}
                        path="/test"
            />
            <Route
                        component={[Function]}
                        path="/home"
            />
            <Route
                        component={[Function]}
                        path="/Meal"
            />
            <Route
                        component={[Function]}
                        path="/userMenu"
            />
            <Route
                        component={[Function]}
                        path="/adminOrders"
            />
            <Route
                        component={[Function]}
                        path="/adminMenu"
            />
            <Route
                        component={[Function]}
                        path="/adminlogin"
            />
            <Route
                        component={[Function]}
                        path="/adminregister"
            />
            <Route
                        component={[Function]}
                        path="/day"
            />
            <Route
                        component={[Function]}
                        path="/now"
            />
            <Route
                        component={[Function]}
                        path="/history"
            />
            <Route
                        component={[Function]}
            />
</Switch>,
          "forceRefresh": true,
          "history": Object {
            "action": "POP",
            "block": [Function],
            "createHref": [Function],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "length": 1,
            "listen": [Function],
            "location": Object {
              "hash": "",
              "pathname": "/",
              "search": "",
              "state": undefined,
            },
            "push": [Function],
            "replace": [Function],
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Route
                component={[Function]}
                exact={true}
                path="/"
/>,
              <Route
                component={[Function]}
                exact={true}
                path="/signup"
/>,
              <Route
                component={[Function]}
                path="/login"
/>,
              <Route
                component={[Function]}
                path="/addMeal"
/>,
              <Route
                component={[Function]}
                path="/updateMeal"
/>,
              <Route
                component={[Function]}
                path="/getMeals"
/>,
              <Route
                component={[Function]}
                path="/deleteMeal"
/>,
              <Route
                component={[Function]}
                path="/getMenu"
/>,
              <Route
                component={[Function]}
                path="/createMenu"
/>,
              <Route
                component={[Function]}
                path="/createOrder"
/>,
              <Route
                component={[Function]}
                path="/getOrder"
/>,
              <Route
                component={[Function]}
                path="/test"
/>,
              <Route
                component={[Function]}
                path="/home"
/>,
              <Route
                component={[Function]}
                path="/Meal"
/>,
              <Route
                component={[Function]}
                path="/userMenu"
/>,
              <Route
                component={[Function]}
                path="/adminOrders"
/>,
              <Route
                component={[Function]}
                path="/adminMenu"
/>,
              <Route
                component={[Function]}
                path="/adminlogin"
/>,
              <Route
                component={[Function]}
                path="/adminregister"
/>,
              <Route
                component={[Function]}
                path="/day"
/>,
              <Route
                component={[Function]}
                path="/now"
/>,
              <Route
                component={[Function]}
                path="/history"
/>,
              <Route
                component={[Function]}
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "exact": true,
                "path": "/",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "exact": true,
                "path": "/signup",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/login",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/addMeal",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/updateMeal",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/getMeals",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/deleteMeal",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/getMenu",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/createMenu",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/createOrder",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/getOrder",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/test",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/home",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/Meal",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/userMenu",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/adminOrders",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/adminMenu",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/adminlogin",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/adminregister",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/day",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/now",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/history",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
